code/util/fetch_data_docDB.py

```diff
@@ -196,6 +206,7 @@ def map_record_to_dict(record):
         'injections': str(injections),
         'task_type': task_type, 
         'virus':virus_names, 
+        'has_fib_in_data_streams': has_fib_in_data_streams,
```

Comment:
comment only– would the idea be when we have video, we'll add a column here for 'has_video_in_data_streams'? 

just wanted to check if the design choice is future-proofed.

Reply:
Yes, the idea is to add `has_xxx` for other modalities, as you suggested in [this issue](https://github.com/AllenNeuralDynamics/foraging-behavior-browser/issues/94). 

Here I'm using `has_video_in_data_streams` to make it explicit that this modality info is from `data_streams`. As discussed [here](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/1056#issuecomment-2495197666), this is a temporary workaround until [David fixes the `data_description`](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/1089), which should eventually be the right place to get modality information.

Reply:
OK fine by me.